documentation/site/pages/zkc/staking.mdx

```diff
@@ -0,0 +1,47 @@
+# Staking
+
+Token holders can stake $ZKC to earn passive staking rewards, as well as participate in governance. 
+
+## Staking $ZKC using the Boundless CLI
+
+TODO
+
+## Staking Rewards
+
+Staking rewards are earned by token holders who have staked $ZKC. These rewards are distributed every reward epoch. $ZKC staking rewards are emitted every reward epoch. Each epoch lasts 2 days, resulting in 182.5 epochs per year.
+
+### Emission Rate
+
+The initial supply is 1 billion ZKC. Emissions are divided equally across epochs throughout the year. The annual emission rate decreases over time based on the years since token launch:
+
+* Year 0: ~7.0% annual  
+* Year 1: ~6.5% annual  
+* Year 2: ~6.0% annual  
+* Year 3: ~5.5% annual  
+* Year 4: ~5.0% annual  
+* Year 5: ~4.5% annual  
+* Year 6: ~4.0% annual  
+* Year 7: ~3.5% annual  
+* Year 8+: ~3.0% annual (minimum rate)
+
+### Emissions Distribution
+
+Emissions occur at the end of each epoch. Every epoch's emissions are allocated to two external contracts for distribution:
+
+#### Passive Pool
+
+25% of epoch emissions are divided proportionately to stakers. For example, if you stake 10% of the $ZKC in the passive staking pool, you will receive 2.5% of the total emissions i.e. 10% of 25%.
+
+#### Active Pool
+
+- A maximum of 75% of epoch emissions are given to provers who have generated proofs during the epoch. 
+- Provers receive a reward for proving based on the percentage of proving (this is tracked via the number of cycles) they have carried out during the epoch. 
+- Provers must stake $ZKC to be eligible for these rewards. Each prover has a reward cap based on their staked $ZKC. This reward cap is set to `staked amount / 15`. 
+    - Therefore if a prover has staked 150 $ZKC, they are eligible to receive a maximum of 10 $ZKC every epoch. If a prover would have received more $ZKC due to their cycle contribution but under-staked $ZKC, the delta remains unclaimed which can be distributed at a later date.
+    - The scalar of 15 was chosen to keep provers aligned with the long term success of the protocol while keeping the barrier entry to proving low to keep the market as open as possible. Future work is to be done to make this scalar more dynamic that encompasses these values.
+
+## Staking Mechanics
+
+- Staking locks $ZKC tokens and mints a veZKC NFT representing an active position. Adding further stake increases the ZKC amount for that active position.
+- Initiating withdrawal starts the 30-day withdrawal period and causes governance voting power and staking rewards to drop to 0 immediately. This is a common mechanism to prevent attacks on governance and the staking reward by timing withdrawals around epoch boundaries. After thirty days, the withdrawal can be completed. Once withdrawn, the veZKC NFT is burned and the tokens previously staked become unlocked. It is not possible to add stake to a position that has already initiated a withdrawal.
```

Comment:
Not sure what you mean about timing around epoch boundaries- its really designed to prevent you from selling before the impact of a governance vote youve participated in takes place. Maybe something like:

This is a common mechanism to prevent attacks on governance and the staking reward by timing withdrawals around epoch boundaries.

=>

This mechanism prevents governance and reward claim attacks, by ensuring that participants in either are financially aligned to the protocol for at least another 30 days.